### DISCUSSION TITLE
How can I improve object detection?

### DISCUSSION THREAD

**Discussion by totom850** [danielgatis/rembg#648] [Q&A]
Hi all!
So I have a frame with wooden plank on it. And I want rembg to remove everything in the frame instead of this plank. But it fails. How can I improve plank detection in my situation? I tried cropping the image, to make it so there is less clutter, but it did not work. Maybe I am doing something wrong?
The image I use:
![image](https://github.com/danielgatis/rembg/assets/147265443/0dc8ca5b-744e-4bf4-9287-0889140daf41)

The result I get:
![1234output](https://github.com/danielgatis/rembg/assets/147265443/28424870-0bc6-4374-906d-061879c48c5d)

**Comment by blagodarren** [comment]
Hi I have exactly the same issue, can we please get support on this?

**Reply by blagodarren** [reply]
can anyone from support please get back to us?

**Comment by Katerinadaisy** [comment]
Hey, I have same problem, would like to hear what is possible solution?
